### ISSUE TITLE
Draft reporting example for unregistered organization for candidates 

### ISSUE THREAD

**Issue by kathycarothers** [fecgov/fec-cms#5114]
### Summary

**What we're after:**
_To help users understand how to report campaign finance transactions, we need to continue drafting the content portion of the reporting examples (language + images) that will go with the candidate section within Help for Candidates and Committees._

- Reporting card for unregistered organization - Candidates 

### Related issues

- #5046

### Completion criteria

The new Candidate Guide list an example in Chapter 10 section 1 under "Contributions from unregistered organizations"

- [ ] Assign reporting example to a RAD analyst volunteer
- [ ] Submit request for images from Jim Jones
- [ ] Draft submitted for review

### Future work

Final review by Brad Matheson, Content Team, and wagtail publication.

**Comment by bmathesonFEC** [comment]
Received draft from RAD Analyst. After review, I'll circulate among the content team and request request images from Jim.

**Comment by bmathesonFEC** [comment]
In addition, the analyst drafted a version for Form 3X filers. I hope to circulate/publish these together.

**Comment by bmathesonFEC** [comment]
Received images from Jim, will have drafts ready by end of this week.

**Comment by bmathesonFEC** [comment]
Updated image, incorporated content team feedback. Waiting on final approval to publish.

**Comment by bmathesonFEC** [comment]
Published the page: https://www.fec.gov/help-candidates-and-committees/filing-reports/contributions-unregistered-organizations/

Updated links here:
- Filing candidate reports: https://www.fec.gov/help-candidates-and-committees/filing-reports/
- All Reporting Examples landing page: https://www.fec.gov/help-candidates-and-committees/reporting-examples/

Added Reporting Example Cards:
- https://www.fec.gov/help-candidates-and-committees/candidate-taking-receipts/questionable-contributions/
- https://www.fec.gov/help-candidates-and-committees/candidate-taking-receipts/who-can-and-cant-contribute/
